src/main/java/core/basesyntax/Classes.java

```diff
@@ -1,9 +1,9 @@
 package core.basesyntax;
 
 public enum Classes {
```

Comment:
Enum names must be singular 
And let's make this name more informative. What kind of classes? Your project can grow and it can be challenging to understand how to use it